src/Bundle/Routing/OperationRouteFactory.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Routing;
```

Comment:
I think that's be better to move this one to Symfony\Component\Resource\Symfony\Routing but we can do that later.